### PR TITLE
Elf

### PR DESCRIPTION
**Pull request by ps3moira**
## About the PR

This PR includes an entire elf-outfit (hat, jumpsuit, shoes) as well as christmas-themed variants of syndicate items like a candycane c20 that shoots candycanes, gingerbread man c4 and bauble hand-grenades.

## Why / Balance

Santa needs some helpers during the holiday season.

## Media
icon
equipped-HELMET
icon
icon
icon
icon

- [X] I have added screenshots/videos to this PR showcasing its changes ingame
**Changelog**

:cl:
- add: Added Elf Costume
- add: Added Candy Cane c20
- add: Added Gingerbread C4
- add: Added Baubles hand-grenade

### CODE REVIEW

File: Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Bombs/gingerbreadc4.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+﻿- type: entity
+  name: ginger-bomb man
+  description: Used to put holes in walls for naughty children.
+  parent: BaseItem
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Bombs/gingerbreadc4.yml]: bruh why not parent c4

Comment (ps3moira) [reply Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Bombs/gingerbreadc4.yml]: true, lemme fix that.

Comment (deltanedas) [reply Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Bombs/gingerbreadc4.yml]: and remove any duplicated fields

Comment (ps3moira) [reply Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Bombs/gingerbreadc4.yml]: > and remove any duplicated fields

what is that?

Comment (deltanedas) [reply Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Bombs/gingerbreadc4.yml]: things like Sticky, triggering on signal, etc
anything the c4 prototype has already you dont need, only keep the differences

Comment (ps3moira) [reply Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Bombs/gingerbreadc4.yml]: Oh right, I can do that.

**Pull request closed without merging**